src/scene/container/container-mixins/effectsMixin.ts

```diff
@@ -92,10 +102,7 @@ export const effectsMixin: Partial<Container> = {
 
         this.effects.splice(index, 1);
 
-        if (this.parentRenderGroup)
-        {
-            this.parentRenderGroup.structureDidChange = true;
```

Comment:
I assume that when effect is removed it should has same logic as `addEffect` for getting rendering group.
Please correct me if I'm wrong.